github/users.go

```diff
@@ -76,6 +76,7 @@ func (u User) String() string {
 // user.
 //
 // GitHub API docs: https://developer.github.com/v3/users/#get-a-single-user
+// GitHub API docs: https://developer.github.com/v3/users/#get-the-authenticated-user
```

Comment:
How about this instead?
```suggestion
// and: https://developer.github.com/v3/users/#get-the-authenticated-user
```